### DISCUSSION TITLE
show bar totals?

### DISCUSSION THREAD

**Discussion by mpat654** [nmalkin/plot-likert#48] [Q&A]
Hello
Thank you for the library. Is there a way to show bar totals for each bar?
Many thanks

**Comment by nmalkin** [comment]
Hi, I'm glad you're finding the library useful!

Could you help me understand where you would want to see the totals for each bar? Is it something like what is done in this StackOverflow post? https://stackoverflow.com/q/51495982

**Comment by mpat654** [comment]
Thanks for the reply!

I actually meant bar totals meaning total counts (i.e number of responses) at the end of each bar
